introduction/installation/sign-up-for-cloud.md

```diff
@@ -1,9 +1,34 @@
 ---
-title: Sign Up for Cloud
+title: Sign Up for GlareDB Cloud
 layout: default
 nav_order: 1
 parent: Installation
 grand_parent: Introduction / Getting Started
 ---
 
-## Sign Up for Cloud
+# Sign Up for GlareDB Cloud
+
+[GlareDB Cloud] is a fully-managed offering of GlareDB that scales with your
+workloads and teams. It is free to get started, and signing up is quick!
+
+![Sign-in]
+
+<!-- TODO: link to pages once they're written -->
+
+After signing up, an Organization with a fully-managed GlareDB Deployment will be
+created and ready to use with the **SQL Workspace** editor in your browser.
```

Comment:
make the sentence active:

Signing up for cloud will create a deployment [what it is] within an organization [what it is].

I might also put the serverless word here.

Reply:
There are comments in the markdown - there are links missing for the Deployment and Organization that will be added.